src/store/messages/api.ts

```diff
@@ -40,3 +40,23 @@ export async function editMessageApi(
 
   return response.status;
 }
+export async function getInfo(): Promise<InfoUploadResponse> {
+  const response = await get<any>('/upload/info');
+  return response.body;
+}
+
+export async function uploadMedia(uploadInfo: InfoUploadResponse, media: File): Promise<any> {
+  const response = await postNative<any>(
+    `${uploadInfo.apiUrl}?timestamp=${uploadInfo.query.timestamp}&signature=${uploadInfo.query.signature}&api_key=${uploadInfo.query.api_key}`
+  ).attach('file', media);
+
+  return response;
+}
+
+export async function uploadFileMessage(channelId: string, media: File): Promise<any> {
+  const response = await post<any>(`/${channelId}/message`)
```

Comment:
I changed this endpoint, it should now be 
```
const response = await post<any>(`/upload/chatChannels/${channelId}/message`).attach('file', media);
```
API PR: https://github.com/m3m3n70/zero-api/pull/860